pages/apis/rest_api/pipelines.md.erb

```diff
@@ -820,6 +820,88 @@ Error responses:
 </tbody>
 </table>
 
+## Unarchive a pipeline
+
+Restore an archived pipeline.
```

Comment:
Unless restore has more meaning than unarchive, lets stick to the same wording so folks don't worry if there is a difference. And it maybe makes sense to reword the description as well

> Unarchived pipeles are editable, and are shown on the Pipeline pages.

Reply:
Otherwise, great stuff!